JobConfig/digitize/OnSpill.fcl

```diff
@@ -5,3 +5,4 @@ physics.producers.EWMProducer.SpillType : 1
 # configure 'Triggered' output to be signal
```

Comment:
The spill type is both set above and in the OnSpill_epilog.fcl, it can likely be removed from one of these places (perhaps from here).

Reply:
good catch, I'll update.